models/servicenow__incident_enhanced.sql

```diff
@@ -66,6 +80,31 @@ incident_enhanced as (
     left join sys_user resolver
         on incident.resolved_by_value = resolver.sys_user_id
         and incident.source_relation = resolver.source_relation
+    left join sys_user caller_info
+        on incident.caller_id_value = caller_info.sys_user_id
+        and incident.source_relation = caller_info.source_relation
+    left join sys_choice category_choice
+        on incident.incident_category = category_choice.sys_choice_value
+        and category_choice.element = 'category'
+        and incident.source_relation = category_choice.source_relation
+    left join sys_choice subcategory_choice
+        on incident.incident_subcategory = subcategory_choice.sys_choice_value
+        and subcategory_choice.element = 'subcategory'
+        and incident.incident_category = subcategory_choice.dependent_value 
+        and incident.source_relation = subcategory_choice.source_relation
+    left join sys_choice state_choice
+        on incident.incident_state = state_choice.sys_choice_value
+        and state_choice.element = 'state'
+        and incident.source_relation = state_choice.source_relation
+    left join sys_choice business_impact_choice
+        on incident.business_impact = business_impact_choice.sys_choice_value
+        and business_impact_choice.element = 'business_impact'
+        -- none when filtered for incident
```

Comment:
Ah yeah so I found that this label doesn't exist in our sys_choice table for this set of filters. Being as this is the only occurrence, I just assume this may be just due to a limitation on our end. I should remove that comment

![image](https://github.com/user-attachments/assets/803f0e66-cd02-40e3-9b38-e54b96d5ab7b)